src/pyscript/plugins/run.py

```diff
@@ -34,6 +34,7 @@ def end_headers(self):
             self.send_header("Cross-Origin-Opener-Policy", "same-origin")
             self.send_header("Cross-Origin-Embedder-Policy", "require-corp")
             self.send_header("Cross-Origin-Resource-Policy", "cross-origin")
+            self.send_header("CacheControl", "no-cache, must-revalidate")
```

Comment:
Should this be `Cache-Control`?

Reply:
[Yes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control)

Reply:
pushed a fix